package.json

```diff
@@ -52,7 +52,7 @@
     "rand-token": "0.4.0",
     "sendgrid": "5.2.3",
     "should": "^13.2.3",
-    "swagger": "git://github.com/swagger-api/swagger-node.git#fffa399",
+    "swagger": "https://github.com/swagger-api/swagger-node.git#fffa399",
```

Comment:
for me "swagger" package is never used. Only uninstall the dependency

Reply:
Ok